docs/modules/ROOT/pages/api/introspection.adoc

```diff
@@ -69,8 +69,8 @@ SRC5 contract implementation extending xref:ISRC5[`ISRC5`].
 * xref:#SRC5-deregister_interface[`++deregister_interface(self, interface_id)++`]
 --
 
-[#SRC5-External-Functions]
-==== External Functions
+[#SRC5-Embeddable-Implementations-Functions]
+==== Embeddable Implementations Functions
```

Comment:
Regarding the title: `Embeddable Implementations Functions`, it reads kind of awkward. This is nitpicky, I know. I don't have a good alternative, just throwing it out there

Reply:
Let's call it just Embeddable Functions in the meantime (I think it reads better). Any suggestion is more than welcome.